CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.28.0 / 2021-06-21
 
+* [CHANGE] Move storage.tsdb.retention.size out of experimental #8728
```

Comment:
Please do not modify the changelog, especially since 2.28 has been released already.

Reply:
Removed this change :)